main/src/ca/uwaterloo/flix/language/phase/monomorph/Lowering.scala

```diff
@@ -315,14 +323,44 @@ object Lowering {
   /**
     * Lowers the given JvmMethod `method`.
     */
-  private def lowervmMethod(method: LoweredAst.JvmMethod)(implicit ctx: Context, root: LoweredAst.Root, flix: Flix): MonoAst.JvmMethod = method match {
+  private def lowerJvmMethod(method: LoweredAst.JvmMethod)(implicit ctx: Context, root: LoweredAst.Root, flix: Flix): MonoAst.JvmMethod = method match {
     case LoweredAst.JvmMethod(ident, fparams, exp, retTyp, eff, loc) =>
       val fs = fparams.map(lowerFormalParam)
       val e = lowerExp(exp)
       val t = lowerType(retTyp)
       MonoAst.JvmMethod(ident, fs, e, t, eff, loc)
   }
 
+  /**
+    * Lowers the given catch rule `rule0`.
+    */
+  private def lowerCatchRule(rule: LoweredAst.CatchRule)(implicit ctx: Context, root: LoweredAst.Root, flix: Flix): MonoAst.CatchRule = rule match {
```

Comment:
I guess there is a bit of inconsistency in that we now have both lowerCatchRule and visitMatchRule